scripts/cloudbuild.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+- name: gcr.io/cloud-builders/docker
+  args: ['build', '--tag=gcr.io/$PROJECT_ID/bazel-jekyll', 'scripts']
```

Comment:
Did you mean:

```
  args: ['build', '--tag=gcr.io/$PROJECT_ID/bazel-jekyll', '//:site']
```

? I see no target called `//:scripts`.

Reply:
No this is a docker build, not a bazel build, so it build the dockerfile in scripts.